# The frame_id maps the audio channel to a sensor device in the URDF.
string frame_id

# the frame_data holds the transported audio data for the channel.
float32[] frame_data